pkg/quota/admission/resourcequota/admission.go

```diff
@@ -54,7 +54,7 @@ func (a *originQuotaAdmission) Admit(as admission.Attributes) error {
 }
 
 func (a *originQuotaAdmission) SetOpenshiftClient(osClient osclient.Interface) {
-	registry := quota.NewRegistry(osClient)
+	registry := quota.NewRegistry(osClient, true)
```

Comment:
@derekwaynecarr - I am the champ at confusing comments.  This isn't related to the `NewRegistry` call. 

The scenario is if I have an admission controller that uses an OpenShift client I cannot fully initialize the controller until it has the client.  So, for example if I need reflectors that are watching OpenShift resources I found myself putting that creation logic in the `SetOpenshiftClient` method.

It would just be nice if there was a life cycle method that could be called that is after any Set* method rather than embedding that logic here.  Holdover from java days I guess, seeing logic unrelated to a field mentioned in the Set* signature smells to me but there is no real harm done.